projects/plugins/boost/app/assets/src/js/utils/generate-critical-css.ts

```diff
@@ -185,7 +201,29 @@ async function generateForKeys(
 			if ( updateResult === false ) {
 				return;
 			}
+
+			// Track individual Critical CSS generation success.
+			const trackerAttributes: TrackerAttributes = {
```

Comment:
I am wondering if it would be useful to also send the URLs for a provider?

Reply:
That data would be useful for debugging, but I'm not sure of the privacy implications of keeping that data. @ebinnion how do we feel about including URLs in tracking data?

Reply:
In this case, since the data is useful for debugging purposes, which will also assist us with helping our users, this seems reasonable to me.